matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/model/annotation/RelationService.kt

```diff
@@ -15,9 +15,13 @@
  */
 package im.vector.matrix.android.api.session.room.model.annotation
 
+import im.vector.matrix.android.api.session.events.model.Event
+import im.vector.matrix.android.api.session.room.model.message.MessageContent
+import im.vector.matrix.android.api.session.room.timeline.TimelineEvent
 import im.vector.matrix.android.api.util.Cancelable
 
-interface ReactionService {
+//TODO rename in relationService?
+interface RelationService {
```

Comment:
From a SDK user point of view, is it clear enough that the RelationService is the service to send reaction, to edit message and to reply to a message? Maybe rename to MessageService or EventService?

Reply:
'Message functionality' is already split in several Services (SendService, ReadService), i have added javadoc to explain the concept of relation

Reply:
ok